src/three-components/SmoothControls.ts

```diff
@@ -34,65 +34,53 @@ export interface SmoothControlsOptions {
   minimumAzimuthalAngle?: number;
   // The maximum angle between model-forward and the camera azimuthal position
   maximumAzimuthalAngle?: number;
-  // The distance from the target orbital position where deceleration will begin
-  decelerationMargin?: number;
-  // The rate of acceleration as the camera starts to change orbital position
-  // The value is measured in world-meters-per-frame-per-frame
-  acceleration?: number;
-  // A scalar in 0..1 that changes the dampening factor, corresponding to
-  // factors from 0.05..0.3
-  dampeningScale?: number;
+  // The minimum camera field of view in degrees
+  minimumFov?: number;
+  // The maximum camera field of view in degrees
+  maximumFov?: number;
   // Controls when events will be cancelled (always, or only when handled)
   eventHandlingBehavior?: EventHandlingBehavior;
   // Controls when interaction is allowed (always, or only when focused)
   interactionPolicy?: InteractionPolicy;
 }
 
 export const DEFAULT_OPTIONS = Object.freeze<SmoothControlsOptions>({
-  minimumRadius: 8,
-  maximumRadius: 32,
+  minimumRadius: 1,
+  maximumRadius: 2,
   minimumPolarAngle: Math.PI / 8,
   maximumPolarAngle: Math.PI - Math.PI / 8,
   minimumAzimuthalAngle: -Infinity,
   maximumAzimuthalAngle: Infinity,
-  decelerationMargin: 0.25,
-  acceleration: 0.15,
-  dampeningScale: 0.5,
```

Comment:
The `Damper` helper seems reasonable, but it would be nice to make some of these qualities configurable in time. It was missing an issue, so I filed https://github.com/GoogleWebComponents/model-viewer/issues/580

See also this earlier discussion on the topic: https://github.com/GoogleWebComponents/model-viewer/issues/498#issuecomment-486709830

For now, YAGNI and thanks for cleaning all of this stuff up 🙇

Reply:
Yeah, I figured that as well; at least now we only need one parameter. For a future PR...